nixos/modules/services/mail/mailman.nix

```diff
@@ -311,9 +311,14 @@ in {
       socketConfig.ListenStream = "/run/mailman-web.socket";
     };
     systemd.services = {
+      postfix = lib.mkIf cfg.enablePostfix {
+        after = [ "mailman.service" ];
```

Comment:
Would it make more sense to put `before = ["postfix.service"];` on mailman?

Reply:
Oh, yes it would.  I'll change that.